dashboard/lib/sqs/queue_processor_config.rb

```diff
@@ -0,0 +1,73 @@
+require_relative '../../../lib/dynamic_config/dcdo'
```

Comment:
Nit: I think this can just be `require 'dynamic_config/dcdo'` (looking at [application_controller.rb#L1](https://github.com/code-dot-org/code-dot-org/blob/52fdd5cd29a83220758d27c39ca8cc2cda2449d4/dashboard/app/controllers/application_controller.rb#L1)).